lib/ingestor-api/runtime/src/validators.py

```diff
@@ -24,14 +24,14 @@ def s3_object_is_accessible(bucket: str, key: str):
     """
     Ensure we can send HEAD requests to S3 objects.
     """
-    from .config import settings
 
-    client = boto3.client("s3", **get_s3_credentials())
+    #  client = boto3.client("s3", **get_s3_credentials())
+    client = boto3.client("s3")
```

Comment:
Why this change? The intention is that we should test access via the data access role, not as the role of this service.  This way, there is a single role that can be used by many different services for data access.  In the event that we need to ask data providers to grant access to our system, it reduces the burden to ask them to grant access to just one role that we use throughout our system.

```suggestion
    client = boto3.client("s3", **get_s3_credentials())
```

Reply:
@alukach If you recall, the Lambda role assumption was failing when we performed initial testing of the construct (see PR comment https://github.com/developmentseed/cdk-pgstac/pull/8#issue-1555544184).  This is a temporary fix so that we can have a functional stack with public buckets until we can diagnose the issue.  There are a few discussions of this on Slack as well https://developmentseed.slack.com/archives/C03N2CN2YJ0/p1670378666706349.